assets/src/apps/authoring/HeaderNav.tsx

```diff
@@ -7,8 +17,64 @@ interface HeaderNavProps {
 
 const HeaderNav: React.FC<HeaderNavProps> = (props: HeaderNavProps) => {
   const { content, isVisible } = props;
+  console.log(content);
   return (
-    <nav className={`aa-header-nav top-panel${isVisible ? ' open' : ''}`}>{content.title}</nav>
+    <nav
+      className={`aa-header-nav top-panel${
+        isVisible ? ' open' : ''
+      } d-flex justify-content-between`}
+      style={{ alignItems: 'center' }}
+    >
+      <div className='col-4'>
+        <div className="btn-group" role="group">
+          <div className="pr-3 border-right">
+            INFINISCOPE{/* Place for Logo */}
+          </div>
+          <div className="pl-3">
+            {content.title}
+          </div>
+        </div>
+      </div>
+      <div className="col-7 btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
```

Comment:
i think you can safely delete the `col-*` from these divs that are children of the `<nav>`. they are causing the header to show scrollbars the moment the screen is resized...